src/react-native-walkthrough-tooltip.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'react-native-walkthrough-tooltip' {
+  import React from 'react';
+  import { GestureResponderEvent, ViewStyle } from 'react-native';
+
+  export interface TooltipProps {
```

Comment:
Why interface here as loggedInUserType is using type

Reply:
loggedInUserType was not created by me. I'll create a separate PR for it.